dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <TheHeader v-if="!useMobileView" />
   <TheContent class="paper-section flex">
-    <CompanyInfoSheet :company-id="companyId" />
+    <CompanyInfoSheet :company-id="companyId" @fetched-company-information="getCompanyName" />
     <div class="card-wrapper">
       <div class="card-grid">
```

Comment:
Can you adapt the stlying of the dialog to be more in line with the figma?
Figma:
![image](https://github.com/d-fine/Dataland/assets/104447806/8688058f-5983-4a04-928e-88a9d1f1f3eb)
Implemented:
![Uploading image.png…]()

Reply:
adapted, I don't know how to get the gray background for the textfield. Do you know how to do that?

Reply:
now it did. I was in the wrong layer

Reply:
ok